core/new-gui/src/app/workspace/types/workflow-common.interface.ts

```diff
@@ -17,6 +17,7 @@ export interface OperatorPredicate extends Readonly<{
   operatorID: string;
   operatorType: string;
   operatorProperties: Readonly<{[key: string]: any}>;
+  operatorGroupName: string;
```

Comment:
revert this change and all other changes related to `operatorGroupName`
the `operatorGroupName` information can be retrieved from `operatorMetadataService.getOperatorSchema(operatorType).additionalMetadata.operatorGroupName`. 

It's good to store a piece of information only at one place. This change creates a redundancy of information.